middleware.ts

```diff
@@ -4,18 +4,28 @@ import type { NextRequest } from 'next/server';
 
 //eslint-disable-next-line
```

Comment:
Ei kannata disabloida eslinttiä kokonaan seuraavalta riviltä, vaan vain tuo yksi ongelmallinen sääntö.
```suggestion
// eslint-disable-next-line @typescript-eslint/require-await
```